Update ne-netadapter-net_packet_filter_flags.md (#1619)
MicrosoftDocs/windows-driver-docs-ddi: wdk-ddi-src/content/netadapter/ne-netadapter-net_packet_filter_flags.md

An initialized [**NET_ADAPTER_RECEIVE_FILTER_CAPABILITIES**](ns-netadapter-net_adapter_receive_filter_capabilities.md) structure is an input to [**NetAdapterSetReceiveFilterCapabilities**](nf-netadapter-netadaptersetreceivefiltercapabilities.md).

> **Important:** If the driver does not report support for all of the packet filters expected by an upper layer, NetAdapterCx will fail the `OID_GEN_CURRENT_PACKET_FILTER` request. This will cause the upper layer will fail to bind to your driver. For reliable binding, your driver must report support for all packet filter types.

Reviewer: ```suggestion
> [!IMPORTANT]
> If the driver doesn't report support for all of the packet filters expected by an upper layer, NetAdapterCx will fail the **OID_GEN_CURRENT_PACKET_FILTER** request, which causes the upper layer to fail to bind to your driver. For reliable binding, your driver must report support for all packet filter types.
```
